.gcb/builds/resources/main.tf

```diff
@@ -46,6 +50,14 @@ resource "google_storage_bucket" "build-cache" {
   }
 }
 
+# Create a Firestore database for the integration tests.
```

Comment:
Do we need instructions for creating the database here:

https://github.com/googleapis/google-cloud-rust/blob/main/doc/contributor/howto-guide-set-up-development-environment.md#one-time-set-up

Reply:
Done.